packages/m/msedit/package.yml

```diff
@@ -0,0 +1,53 @@
+name       : msedit
+version    : 1.2.1
+release    : 1
+source     :
+    - https://github.com/microsoft/edit/archive/refs/tags/v1.2.1.tar.gz : 41c719b08212fa4ab6e434a53242b2718ba313e8d24d090f244bb857d6a9d0fd
+homepage   : https://github.com/microsoft/edit
+license    : MIT
+component  : system.utils
+networking : true
+summary    : Microsoft Edit - A simple editor for simple needs
+description: |
+    This editor pays homage to the classic MS-DOS Editor, but with a modern
+    interface and input controls similar to VS Code. The goal is to provide an
+    accessible editor that even users largely unfamiliar with terminals can
+    easily use.
+
+builddeps  :
+    - git
+    - rustup
+
+setup      : |
+    rustup update stable
```

Comment:
Please use `rust` as the build dependency, and don't try to update the toolchain.

Reply:
It's going to have to wait, then, yeah.